MobSF/utils.py

```diff
@@ -59,8 +60,17 @@ def check_update():
     try:
         print("\n[INFO] Checking for Update.")
         github_url = "https://raw.githubusercontent.com/MobSF/Mobile-Security-Framework-MobSF/master/MobSF/settings.py"
-        response = urllib.request.urlopen(github_url)
-        html = str(response.read()).split("\n")
+        if settings.UPSTREAM_PROXY_ENABLED:

```

Comment:
We should prolly move this repeating function into a single function in `utils.py` and call it with a url and  get the response. Maintenance will be easy and code  will look neat as we avoid redundancy. I am good with this, but do this only if you are comfortable.